src/unitxt/templates.py

```diff
@@ -528,6 +537,17 @@ def prepare(self):
                 "XX",
             ]
 
+    def verify(self):
+        super().verify()
+        print(self.postprocessors[-1])
```

Comment:
I think these checks can be removed. processors.match_closest_option post processor is not a requirement of the MultipleChoiceTemplate.